auth-server/src/utils/validateName.ts

```diff
@@ -8,7 +8,7 @@ import messages from './messages';
 
 export default (name: string): void => {
 	// empty name are accepted
-	if (name === '') {
+	if (!name || name === '') {
```

Comment:
Not sure where this is needed.

Reply:
removed. Was used to allow null name. Not needed